index.html

```diff
@@ -0,0 +1,280 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hello Microverse act </title>
+  <link rel="stylesheet" href="./css/styles.css" />
+  <script
+  src="https://kit.fontawesome.com/716817a6e7.js"
+  crossorigin="anonymous"
+></script>
+</head>
+
+<body>
+ <!-- Header Section -->
+ <header class="header">
+  <div class="header-container">
+    <a href="#" class="logo animation-rotate">Geegee</a>
+    <nav class="nav-bar">
+      <ul class="navigation-bar nav-menu">
+        <li><a href="#portfolio" class="list-item">Portfolio</a></li>
+        <li><a href="#about" class="list-item">About</a></li>
+        <li><a href="#contact" class="list-item">Contact</a></li>
+      </ul>
+    </nav>
+    <div class="hamburger">
```

Comment:
- Kindly as this is a mobile version, make the hamburger button visible only until desktop resolutions (up to 768px)

Reply:
@BregornOriginal Done.  thanks, buddy.
I just started following your GitHub account.
Thanks for the advice